### PR TITLE
BUILD-2320 Migrate to Cirrus CI on AWS

### PR DESCRIPTION
**Pull request by elrzn**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -1,10 +1,5 @@
-gcp_credentials: ENCRYPTED[!149d4005ecdba4cdd78bb5ba22756ebb98bf8e3367ee2e9ab08c5a1608c0d3e3b501904b67a1d67c0b63085e469d7dde!]
- 
 env:
   CIRRUS_CLONE_DEPTH: 20
-  CIRRUS_VAULT_AUTH_PATH: jwt-cirrusci
```
Comment (vlad-sonar) [inline .cirrus.yml:5]: why do you remove CIRRUS env here?
I haven't seen this as part of migration to AWS before ))

Comment (elrzn) [reply .cirrus.yml:5]: It is provided here https://github.com/SonarSource/cirrus-modules/blob/master/features/vault.star#L14

Which is loaded by default.

Comment (vlad-sonar) [reply .cirrus.yml:5]: I see, thanks for the explanation. To me, this should be done as part of the migration to Vault.
David said that Evgeny did not want to use cirrus.star, which explains a lot.
Can you please double-check, if that's the case?

Comment (Godin) [reply .cirrus.yml:5]: > David said that Evgeny did not want to use cirrus.star

I never said this

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -5,9 +5,6 @@ on:
     types:
       - published
 
-env:
-  PYTHONUNBUFFERED: 1
-
```
Comment (Godin) [inline .github/workflows/release.yml:10]: This change is unrelated to BUILD-2320, next time will be better to do it via separate PR.

**Review by Godin — APPROVED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (drautureau-sonarsource) [reply .cirrus.yml:5]: > > David said that Evgeny did not want to use cirrus.star
> 
> I never said this

Me neither, that was a guess
